todo_app/app.py

```diff
@@ -1,36 +1,67 @@
 from flask import Flask, render_template, url_for, redirect, request
-from todo_app.data.session_items import get_items, add_item, save_item, get_item, remove_item
+from todo_app.data.session_items import get_item
+from todo_app.data.trello_items import get_items, add_item, change_item_status, remove_item, Item, get_list_names
+from dotenv import load_dotenv
+import os
 
 
 from todo_app.flask_config import Config
 
+load_dotenv()
+
+board_id = os.getenv("BOARD_ID")
+apikey = os.getenv("APIKEY")
+token = os.getenv("TOKEN")
+
 app = Flask(__name__)
 app.config.from_object(Config())
 
+list_targets = get_list_names(board_id, apikey, token)
```

Comment:
I think API Key should be two words, as such please can you change it to be `api_key` here, and make a similar change everywhere else you have a variable called apikey or APIKEY.